src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -91,12 +91,15 @@
        :div {:style {:position (when-not sidebar-visible? "fixed")
                      :top (when-not sidebar-visible? 0)
                      :width 270}}
-       (when false ; curator? TODO commented out until ready
+       (when (and curator? owner?)
```

Comment:
to edit metadata of a Library dataset, you do not have to be an owner - you can be a writer (clarified with Moran a few days ago)